### PR TITLE
Add support for compressed LAZ files

### PR DESCRIPTION
**Pull request by jgrocha**
This PR closes issue #845. It:
* adds LASzip library to docker. 
* updates PDAL version
* changed pdal pipeline las.json used to write the `*.laz` point cloud

Thank you @smathermather and @pierotofy for the guidance.

### CODE REVIEW

File: opendm/types.py [hunk 1]
```diff
@@ -478,7 +479,7 @@ def __init__(self, root_path, images_path, gcp_file = None):
         self.odm_georeferencing_las_json = io.join_paths(
             self.odm_georeferencing, 'las.json')
         self.odm_georeferencing_model_las = io.join_paths(
```
Comment (pierotofy) [inline opendm/types.py]: Should we rename this variable to `odm_georeferencing_model_laz`?

**Review by pierotofy — APPROVED** [review]

File: opendm/types.py (hunk 1 above)
Comment (smathermather) [reply opendm/types.py]: For consistency, I think so.

**Review by smathermather — APPROVED** [review]

**Review by smathermather — APPROVED** [review]

**Comment by jgrocha** [comment]
Thank you @smathermather

**Comment by smathermather** [comment]
Standby. Making a Monday morning mess over here... .

File: opendm/types.py (hunk 1 above)
Comment (jgrocha) [reply opendm/types.py]: I think @pierotofy  was talking about `odm_georeferencing_model_las`.

**Pull request merged**